styleguide/_themes/derek/scss/patterns/blog-feed.scss

```diff
@@ -0,0 +1,71 @@
+@mixin blogFeed-textElements {
+  color: $medium-light-gray;
+  font-size: .9em;
+  font-weight: 400;
+  line-height: 1.5em;
+}
+
+.blogFeed-date,
+.blogFeed-excerpt,
+.blogFeed-author,
+.blogFeed-link,
+.blogFeed-linkIcon {
+  @include blogFeed-textElements;
+}
+
+.blogFeed-item {
+  @include make-md-column(4);
+  margin: 15px 5px;
+}
+
+.blogFeed-imageContainer {
+  @include make-md-column(11);
+  padding: 0;
+}
+
+.blogFeed-image {
+  @include sepia-filter;
+}
+
+.blogFeed-textContainer {
+  @include make-md-column(11);
+  background-color: $white;
+}
+
+.blogFeed-date {
+  padding-top: 15px;
+}
+
+.blogFeed-title {
+  color: $medium-dark-gray;
+}
+
+.blogFeed-author {
+  font-size: .7em;
+
+  &::before {
+    content: 'By: ';
```

Comment:
Maybe not the very first use of this but l10n should always be up front. We can just pull it from css and prepend it in Drupal